### PR TITLE
plugin.audio.tunir - v0.0.5

### PR DESCRIPTION
**Pull request by nire0510**
New plugin - plugin.audio.tunir
Collection of more than 35000 international radio broadcasts

### CODE REVIEW

File: plugin.audio.tunir/addon.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.tunir" name="Tunir" version="0.0.5" provider-name="Nir Elbaz">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
```
Comment (learningit) [inline plugin.audio.tunir/addon.xml]: Please set xbmc.python to 2.24.0 for Jarvis. Thanks.

**Comment by nire0510** [comment]
@learningit done, thanks (y)

**Comment by learningit** [comment]
You need to squash your commits into a single commit. Please see: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by nire0510** [comment]
Thanks a lot, @learningit , will do!

**Pull request merged**